tools/barista/src/builder/components.ts

```diff
@@ -32,13 +38,20 @@ import {
   transformPage,
   uxSlotTransformer,
   headingIdTransformer,
+  exampleInlineSourcesTransformerFactory,
 } from '../transform';
 
 import { slugify } from '../utils/slugify';
 
 const PROJECT_ROOT = join(__dirname, '../../../');
 const LIB_ROOT = join(__dirname, '../../../', 'components');
 const DOCUMENTATION_ROOT = join(__dirname, '../../../', 'documentation');
+const EXAMPLES_METADATA = join(
+  __dirname,
+  '../../../',
```

Comment:
NIT: can we put the `'../../../'` somewhere in a variable in case you are using it often and maybe provide a path to it.

if the file changes the folder because you move it you would have to adapt it everywhere